samples/nrf9160/serial_lte_modem/README.rst

```diff
@@ -95,7 +95,7 @@ Generic commands
 The following proprietary generic AT commands are used in this sample:
 
 * AT#XSLMVER
-* AT#XSLEEP=<mode>
+* AT#XSLEEP
```

Comment:
This was "AT#XSLEEP (only for external MCU configuration)" before the change.
Not sure if the extra information is needed?

Reply:
Based on customer feedback, Sleep/Wakeup is enabled for PC client as well. So this comment does not stand any more.

Reply:
ok